src/parser/view_transform.c

```diff
@@ -1735,7 +1736,7 @@ mq_substitute_subquery_in_statement (PARSER_CONTEXT * parser, PT_NODE * statemen
 	  rewrite_as_derived = true;
 	}
       /* check for CONNECT BY */
-      else if (tmp_result->info.query.q.select.connect_by)
+      else if (PT_IS_SELECT (tmp_result) && tmp_result->info.query.q.select.connect_by)
```

Comment:
it looks not related to this issue.

Reply:
There was a problem with node type and I fixed it together. :)